pkg/hooks/v1alpha2/v1alpha2.go

```diff
@@ -20,3 +20,4 @@
 package v1alpha2
 
 const Version = "v1alpha2"
+const Deprecated = true
```

Comment:
This now also deprecates `v1alpha2`, not only `v1alpha1`. I think the PR description needs an update to reflect that. Especially the release note (currently it only mentions `v1alpha1`).

Reply:
Fixed